src/examples/graph.js

```diff
@@ -532,17 +641,43 @@ class Graph extends React.Component<IGraphProps, IGraphState> {
               ))}
             </select>
           </div>
+          <div className="allow-multiselect">
+            <label>
+              Multiselect:
```

Comment:
Should this be "Multi-select"? (There's a mix of `multiselect` and `multiSelect` in classnames/variables/etc.)

Reply:
Thanks. Updated to Multiselect instead of MultiSelect.